src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/persistence/SQLitePersistor.java

```diff
@@ -52,230 +54,311 @@
 import org.jooq.impl.DSL;
 
 class SQLitePersistor extends PersistorBase {
-  private static final String DB_URL = "jdbc:sqlite:";
-  private DSLContext create;
-  private Map<String, List<Field<?>>> jooqTableColumns;
-  private static final Logger LOG = LogManager.getLogger(SQLitePersistor.class);
-  private static final String LAST_INSERT_ROWID = "last_insert_rowid()";
-  private static final String PRIMARY_KEY_AUTOINCREMENT_POSTFIX = " INTEGER PRIMARY KEY AUTOINCREMENT";
+    private static final String DB_URL = "jdbc:sqlite:";
```

Comment:
indent is incorrect in this file

Reply:
+1, would be good to fix the indent and update the PR to make it easy to read the diff